_i18n/en.yml

```diff
@@ -0,0 +1,81 @@
+---
+global:
+  brazilian portuguese: Português
+  english: English
+header:
+  stories: Stories
+  explore: Explore
+  about: About
+support:
+  fund_us: Support the project
+home:
+  about: Artificial Intelligence for social control of public administration
+  about_rosie: Operação Serenata de Amor created a Robot - An artificial intelligence capable of analysing each request for reimbursement from congresspeople and identify it’s illegal probability.
+  last_histories: Last published histories
+  loading: Loading...
+  more_posts: More posts
+footer:
+  about: about
+  github: Serenata on github
+  contact: contact
+  partners: partners
+stories:
+  published: Published stories
+  loading: Loading...
+explore:
+  reports: Reports
+  about_reports: Rosie, our Robot, in numbers
+  tweet_about: Tweet about this number
+  chamber_reports: Reports done to the Chamber of Deputies
+  suspitions: Different congresspeople reported for suspicious reimbursements
+  questionated: thousand Brazilian Reals in suspicious expenses questioned
```

Comment:
_thousand Brazilian Reals questioned in suspicious expenses_ (instead of _thousand Brazilian Reals in suspicious expenses questioned_)